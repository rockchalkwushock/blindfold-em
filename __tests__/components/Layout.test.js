import React from 'react'
import { render } from 'enzyme'

import Layout from '../../components/layout'

const Test = () => (
  <Layout>
    <div>1</div>
    <div>2</div>
    <div>3</div>
  </Layout>
)

describe('Component: <Layout />', () => {
  test('should render without exploding', () => {
    const tree = render(<Test />)
    expect(tree).toMatchSnapshot()
  })
})
