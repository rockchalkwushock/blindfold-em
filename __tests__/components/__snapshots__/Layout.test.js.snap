// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Layout /> should mount without exploding 1`] = `
<Test>
  <Layout>
    <FlexContainer
      className="layout"
    >
      <div
        className="jsx-1664832883 layout"
      >
        <Component>
          <div
            className="jsx-2859325892"
          >
            <SideEffect(Head)>
              <Head />
            </SideEffect(Head)>
            <JSXStyle
              css="*{box-sizing:border-box;-webkit-box-sizing:border-box;margin:0;padding:0;}html,body{min-height:100vh;overflow-x:hidden;}body{background-color:#A974C2;color:#B0BF6C;font-family:\\"Kalam\\", cursive;font-size:1em;line-height:1.5em;}"
              styleId="2859325892"
            />
          </div>
        </Component>
        <Header>
          <header
            className="jsx-3420915757"
          >
            Blindfold'em
            <JSXStyle
              css="header.jsx-3420915757{-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;background-color:#3A2A41;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;font-family:\\"Permanent Marker\\", cursive;height:20vh;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-letter-spacing:0.3em;-moz-letter-spacing:0.3em;-ms-letter-spacing:0.3em;letter-spacing:0.3em;padding:1em;text-align:center;}@media (min-width:425px){header.jsx-3420915757{font-size:1.25em;}}@media (min-width:768px){header.jsx-3420915757{font-size:1.75em;-webkit-letter-spacing:0.5em;-moz-letter-spacing:0.5em;-ms-letter-spacing:0.5em;letter-spacing:0.5em;}}"
              styleId="3420915757"
            />
          </header>
        </Header>
        <FlexContainer
          className="main"
        >
          <div
            className="jsx-1664832883 main"
          >
            <div>
              1
            </div>
            <div>
              2
            </div>
            <div>
              3
            </div>
            <JSXStyle
              css="div.jsx-1664832883{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;}.layout.jsx-1664832883{margin:auto;max-width:800px;min-height:100vh;}.main.jsx-1664832883{background-color:#734D84;height:65vh;padding:0.5em;}.wizard.jsx-1664832883{-webkit-flex:auto;-ms-flex:auto;flex:auto;}.completed.jsx-1664832883,.frame.jsx-1664832883,.timer.jsx-1664832883{-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;background-color:#3A2A41;box-shadow:4px 4px 4px #A974C2;-webkit-flex:auto;-ms-flex:auto;flex:auto;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;padding:0 0.5em;text-align:center;}@media (orientation:landscape){.main.jsx-1664832883{height:80vh;padding:1em;}}@media (min-width:768px){.main.jsx-1664832883{padding:2em 3em;}}"
              styleId="1664832883"
            />
          </div>
        </FlexContainer>
        <Footer>
          <footer
            className="jsx-1161495840"
          >
            <JSXStyle
              css="footer.jsx-1161495840{background-color:#3A2A41;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;height:15vh;padding:1em;}"
              styleId="1161495840"
            />
          </footer>
        </Footer>
        <JSXStyle
          css="div.jsx-1664832883{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;}.layout.jsx-1664832883{margin:auto;max-width:800px;min-height:100vh;}.main.jsx-1664832883{background-color:#734D84;height:65vh;padding:0.5em;}.wizard.jsx-1664832883{-webkit-flex:auto;-ms-flex:auto;flex:auto;}.completed.jsx-1664832883,.frame.jsx-1664832883,.timer.jsx-1664832883{-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;background-color:#3A2A41;box-shadow:4px 4px 4px #A974C2;-webkit-flex:auto;-ms-flex:auto;flex:auto;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;padding:0 0.5em;text-align:center;}@media (orientation:landscape){.main.jsx-1664832883{height:80vh;padding:1em;}}@media (min-width:768px){.main.jsx-1664832883{padding:2em 3em;}}"
          styleId="1664832883"
        />
      </div>
    </FlexContainer>
  </Layout>
</Test>
`;
